ros2_batch_job/__main__.py

```diff
@@ -190,6 +190,9 @@ def get_args(sysargv=None):
     parser.add_argument(
         '--src-mounted', default=False, action='store_true',
         help="src directory is already mounted into the workspace")
+    parser.add_argument(
+        '--compile-on-clang', default=False, action='store_true',
+        help="compile on clang instead of gnu")
```

Comment:
Wording: `gnu` is not the compiler - it is `GCC` or `GNU Compiler Collection`.

Reply:
Makes sense, changed the wording